hello-world-notification-microservice/README.md

```diff
@@ -72,4 +76,69 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+
+## Prerequisite
+- cumulocity tenant with microservice hosting feature enabled
+    - for free trial account on cumulocity.com you will need to request this feature by contacting
+- contents of this repository
+
+### Instructions
+1. Creating a test user
+    - From the Application Switcher select Administration
+    - In the Navigation pane expand Accounts and go to Users
+    - Click on 'Add user', fill in the form for the new user, untick "Send password reset link as email" and provide a password
+    - Click Save
+    - Enable notification support:
+        - Using Navigation pane go to Accounts and then Roles, click on "Add global role"
+        - Name the role "Notifications" and from the list of permissions tick "Notification 2" in Admin column
+        - click Save
+        - go to Accounts then Users
+        - in the list find the username created a few steps before and under "Global roles" open the dropdown
+        - find and tick "Notifications" and click "Apply"
+2. Creating a test device
+    - open the demo directory
+    - edit the `measurements.py` script and provide the following credentials for your newly created user as well as the platform url:
+      ```python
+      _tenant_id = ''
+      _username = ''
+      _password = ''
+      _platform_url = ''
+        ```
```

Comment:
I don't think it's clear what is input/typed by user and what is output from script in the console below. (Same as the outdated comment I made)

As in the ops-docs, should we should the command template in a bold comment first using DEVICE_ID placeholder and then show example of that command in use (with output showing a real - but made up - deviceId) in the console